test/functional/custom-types.json

```diff
@@ -0,0 +1,67 @@
+{
+    "runtime_id": 2,
+    "types": {
+        "DestinationCreatePP": {
+            "type": "struct",
+            "type_mapping": [
+                [ "code", "Vec<u8>" ],
+                [ "data", "Vec<u8>" ]
+            ]
+        },
+        "DestinationCallPP": {
+            "type": "struct",
+            "type_mapping": [
+                [ "dest_account", "AccountId" ],
+                [ "input_data", "Vec<u8>" ]
+            ]
+        },
+        "Destination": {
+            "type": "enum",
+            "type_mapping": [
+                [ "Pubkey", "H256" ],
+                [ "CreatePP", "DestinationCreatePP" ],
+                [ "CallPP", "DestinationCallPP" ],
+                [ "ScriptHash", "H256" ],
+                [ "PubkeyHash", "Vec<u8>" ]
```

Comment:
We don't have `Destionation::PubkeyHash` anymore. I think this file should be checked against `utxo-tester`'s `custom-types.json` as right now it's known to be up to date.